src/sources/forus-platform/pug/tpl/pages/website/contact.pug

```diff
@@ -6,7 +6,7 @@ section
     .container
         .row.wrap_cs__2.no-pad-t.flexed
             .col-6.col-md-6.e-fb.flexed
-                iframe(class="g_maps" src="https://maps.google.com/maps?q=Emmaplein%206%2C%209711%20AP%20Groningen&t=&z=13&ie=UTF8&iwloc=&output=embed")
```

Comment:
@aghimpu could you check if link change is correct? i took it from embed link in google maps, but it's much longer accidentally